src/main/target/FLYWOOF405NANO/target.h

```diff
@@ -0,0 +1,166 @@
+/*
+ * This file is part of INAV.
+ *
+ * INAV is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * INAV is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with INAV.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "F4NA"
+#define USBD_PRODUCT_STRING  "FLYWOOF405NANO"
+
+
+#define LED0                    PC14  //Green
+#define BEEPER                  PC13
+#define BEEPER_INVERTED
+
+// *************** SPI1 Gyro & ACC *******************
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN   	    PA6
+#define SPI1_MOSI_PIN   	    PA7
+
+#define USE_IMU_MPU6000
+#define IMU_MPU6000_ALIGN       CW180_DEG
+#define MPU6000_CS_PIN          PB12
+#define MPU6000_SPI_BUS         BUS_SPI1
+
+#define USE_IMU_BMI270
+#define IMU_BMI270_ALIGN       CW180_DEG
+#define BMI270_CS_PIN          PB12
+#define BMI270_SPI_BUS         BUS_SPI1
+
```

Comment:
`#define USE_IMU_ICM42605
#define IMU_ICM42605_ALIGN      CW90_DEG
#define ICM42605_CS_PIN         PB12
#define ICM42605_SPI_BUS        BUS_SPI1`

Reply:
Hi, this are the config files provided to me by Flywoo

[Flywoo F405NANO.zip](https://github.com/iNavFlight/inav/files/12385464/Flywoo.F405NANO.zip)